<launch>

  <arg name="simulated" default='true'/>
  <arg name="use_rik" default='true' />

  <!-- If using Relaxed-IK -->
  <group if="$(eval arg('use_rik') == true)">
    <include file="$(find iter_app)/launch/ur5-rik.launch">
      <arg name="simulated" value="$(arg simulated)"/>
    </include>
  </group>

  <!-- Else, use moveit-->
  <group if="$(eval arg('use_rik') == false)">
    <include file="$(find iter_app)/launch/ur5-moveit.launch">
      <arg name="simulated" value="$(arg simulated)"/>
    </include>
  </group>

</launch>
